src/backend/api/Fusion.Resources.Domain/Commands/Contracts/UpdateContractReps.cs

```diff
@@ -67,7 +67,7 @@ protected override async Task Handle(UpdateContractReps request, CancellationTok
 
                 if (request.ContractResponsiblePositionId.HasBeenSet)
                 {
-                    if (request.ContractResponsiblePositionId.Value.HasValue && contract.CompanyRep?.Id != request.ContractResponsiblePositionId.Value.Value)
+                    if (request.ContractResponsiblePositionId.Value.HasValue && contract.ContractRep?.Id != request.ContractResponsiblePositionId.Value.Value)
```

Comment:
Assuming the previous CompanyRep was incorrect here.

Reply:
By the if check, the wrong property is used, yes

Reply:
Indeed - that is the bugfix,